charon/tests/ui/unsupported/rename_attribute_failure.rs

```diff
@@ -0,0 +1,17 @@
+//@ known-failure
```

Comment:
The `unsupported` folder is for things we don't support yet but want to support eventually. Could you move this test out of the folder?